templates/voting_platform/vote_step/_candidate_box.html.twig

```diff
@@ -0,0 +1,44 @@
+<div class="candidate__box l__row  b__nudge--bottom-medium">
+    {{ form_widget(form) }}
+    {{ form_label(form, ' ') }}
```

Comment:
should be false no ? otherwise Symfyony will try to translate `' '`, and it willl be missing

Reply:
we use label as a clickable area, is there is another way to do it ?

Reply:
we need to have an invisible `<label>` tag to wrap the box content: 
![image](https://user-images.githubusercontent.com/4582866/81320099-ef239b80-9090-11ea-8eb1-b697bfc4ff51.png)

with `false`, `null` value, form does not display `<label>` tag